pages/index.tsx

```diff
@@ -5,12 +5,21 @@ import {
   Flex,
   HStack,
   Heading,
+  Image,
   Link,
+  Spacer,
   Text,
   VStack,
 } from "@chakra-ui/react";
+import { isMobile } from "react-device-detect";
+import { useEffect } from "react";
 
 export default function Home() {
+  let pageLogoWidth;
+  useEffect(() => {
+    isMobile ? (pageLogoWidth = "40%") : (pageLogoWidth = "15%");
+  }, [isMobile]);
```

Comment:
Why import another library for this? Chakra has an API for responsive design: https://chakra-ui.com/docs/features/responsive-styles 🤓

Reply:
Hmm idk why someone would do that 😆🤣 I'll switch it